scripts/update-models.ts

```
@@ -0,0 +1,103 @@
+
+  const content = `export const ${name} = {\n${entries}\n} as const;\n\nexport type ${name} = (typeof ${name})[keyof typeof ${name}];\n`;
+  fs.writeFileSync(path.resolve(filePath), content);
+  console.log(`Updated ${filePath}`);
+}
+
+async function updateOpenAI(apiKey: string) {
+  const client = new OpenAI({ apiKey });
+  const list = await client.models.list();
+  const ids = list.data.map((m: any) => m.id as string);
+
+  const chat = ids.filter((id) =>
```

ESLint

[error] Replace `(id)` with `⏎····id`

(prettier/prettier)